src/components/Login.js

```diff
@@ -16,7 +18,11 @@ export class Login extends Component {
     handleSubmit = (e) => {
         /* replace the value of v with backend function*/
         const v = this.state.pw === 'password';
-        this.setState({ verified: v })
+        this.setState({ verified: v });
+        this.setState({ pw: '' })
```

Comment:
What is this for?

Reply:
idk why but when i hit submit/enter, the text doesn't clear. So i just manually made it clear by making pw and empty string .

Reply:
Ah right. But shouldn't they not see the form after logged in?

Reply:
yeah i think it's only useful if they get the password wrong, so it'll clear it and they can just type it again instead of backspacing it all and then typing. Do you think i should do smth like if (!verified) {

Reply:
Oh okay cool nah thats fine